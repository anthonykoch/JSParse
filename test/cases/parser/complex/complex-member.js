user.name.first[zero][next]
