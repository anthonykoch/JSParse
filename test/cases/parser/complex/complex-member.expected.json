---
name: complex member
description: complex member expression
skip: true
---

[]
